packages/camera/camera/android/src/test/java/io/flutter/plugins/camera/features/sensororientation/DeviceOrientationManagerTest.java

```diff
@@ -62,13 +62,13 @@ public void getVideoOrientation_whenNaturalScreenOrientationEqualsPortraitUp() {
         deviceOrientationManager.getVideoOrientation(DeviceOrientation.LANDSCAPE_RIGHT);
 
     assertEquals(0, degreesPortraitUp);
-    assertEquals(90, degreesLandscapeLeft);
+    assertEquals(270, degreesLandscapeLeft);
     assertEquals(180, degreesPortraitDown);
-    assertEquals(270, degreesLandscapeRight);
+    assertEquals(90, degreesLandscapeRight);
   }
 
   @Test
-  public void getVideoOrientation_whenNaturalScreenOrientationEqualsLandscapeLeft() {
+  public void getVideoOrientationBackFacing_whenNaturalScreenOrientationEqualsLandscapeLeft() {
```

Comment:
Not sure if having separate tests for `getVideoOrientationBackFacing_xxx` and `getVideoOrientationFrontFacing_xxx` is a good thing? Back/front facing seems to not have an effect on the switch case that was changed.

Reply:
Per an offline discussing with @Code-Remote I removed the front facing tests as those are redundant.